app/mailers/application_mailer.rb

```diff
@@ -13,9 +13,14 @@ def self.should_send_internal_emails?
 
   private
 
-  def add_image(image_file_name)
-    path = Rails.root.join("app/assets/images/mailer/#{image_file_name}")
-    attachments.inline[image_file_name] = File.read(path)
+  def add_image(image_file_name, source = "mailer")
+    if source == "mailer"
+      path = Rails.root.join("app/assets/images/mailer/#{image_file_name}")
+      attachments.inline[image_file_name] = File.read(path)
+    elsif source == "assets"
+      path = Rails.root.join("app/assets/images/#{image_file_name}")
+      attachments.inline[image_file_name] = File.read(path)
+    end
```

Comment:
(minor: if we decided on using the method as is, probably there would be an `else` case which raised an error)